lib/physical/cuboid.rb

```diff
@@ -26,7 +26,7 @@ def density
     end
 
     def ==(other)
-      id == other.id
+      id == other&.id
```

Comment:
I think we should actually have a type check of sorts here. This way, if we have an ID of `1`, this would be equal to any ActiveRecord object that happens to be the first line in a table...

Reply:
Done!